### PR TITLE
Partial update of NL i18n

### PR DESCRIPTION
**Pull request by TychoWerner**

### CODE REVIEW

File: i18n/nl.json [hunk 1]
```diff
@@ -14,10 +14,10 @@
     "bounces.unknownService": "Onbekende service.",
     "bounces.view": "Zie bounces",
     "campaigns.addAltText": "Voeg alternatieve tekst zonder opmaak toe",
-    "campaigns.archive": "Archive",
-    "campaigns.archiveEnable": "Publish to public archive",
-    "campaigns.archiveHelp": "Publish (running, paused, finished) the campaign message on the public archive.",
-    "campaigns.archiveMeta": "Campaign metadata",
+    "campaigns.archive": "Archiveren",
+    "campaigns.archiveEnable": "Publiceren naar publiek archief.",
```
Comment (Jjagg) [inline i18n/nl.json:18]: There's no trailing period in the original English text here.

```suggestion
    "campaigns.archiveEnable": "Publiceren naar publiek archief",
```

File: i18n/nl.json [hunk 2]
```diff
@@ -14,10 +14,10 @@
     "bounces.unknownService": "Onbekende service.",
     "bounces.view": "Zie bounces",
     "campaigns.addAltText": "Voeg alternatieve tekst zonder opmaak toe",
-    "campaigns.archive": "Archive",
-    "campaigns.archiveEnable": "Publish to public archive",
-    "campaigns.archiveHelp": "Publish (running, paused, finished) the campaign message on the public archive.",
-    "campaigns.archiveMeta": "Campaign metadata",
+    "campaigns.archive": "Archiveren",
```
Comment (Jjagg) [inline i18n/nl.json:17]: Sorry for the separate review, but I think this should be the noun archive ('Archief'), not the verb ('Archiveren'). At least that's how it is in the german translations. @knadh?

Comment (knadh) [reply i18n/nl.json:17]: Yep, it should be the noun.

Comment (knadh) [reply i18n/nl.json:17]: @TychoWerner Could you please amend the PR with this change?

Comment (TychoWerner) [reply i18n/nl.json:17]: These changes are valid.

**Pull request merged**